File: Beecrowd-Python/1041.py
# Lê os valores de x e de y
x, y = map(float, input().split())

# Verifica em qual quadrante o ponto se encontra
if x == 0 and y == 0:
    print("Origem")
elif x == 0:
    print("Eixo Y")
elif y == 0:
    print("Eixo X")
elif x > 0 and y > 0:
    print("Q1")
elif x < 0 and y > 0:
    print("Q2")
elif x < 0 and y < 0:
    print("Q3")
else: 
    print("Q4")

File: Beecrowd-C/1041.c
#include <stdio.h>
#include <math.h>

int main() {
    // Declaração de variáveis para armazenar os valores X e Y
    double X, Y;

    // Solicita ao usuário que insira os valores X e Y
    scanf("%lf%lf", &X, &Y);

    // Verifica em qual quadrante ou eixo os valores estão localizados
    if (X > 0.0 && Y > 0.0) {
        printf("Q1\n");
    } 
    else if (X > 0.0 && Y < 0.0) {
        printf("Q4\n");
    }
    else if (X < 0.0 && Y > 0.0) {
        printf("Q2\n");
    }
    else if (X < 0.0 && Y < 0.0) {
        printf("Q3\n");
    }
    else if (X == 0.0 && Y == 0.0) {
        printf("Origem\n");
    }
    else if (X == 0.0) {
        printf("Eixo Y\n");
    }
    else if (Y == 0.0) {
        printf("Eixo X\n");
    }
    
    return 0;
}